### ISSUE TITLE
reminder: Decentraleyes & LocalCDN

### ISSUE THREAD

**Issue by Thorin-Oakenpants** [arkenfox/user.js#948]
meh

**Comment by Thorin-Oakenpants** [comment]
for posterity
![its-dead-jim](https://user-images.githubusercontent.com/16656956/94345730-bd977680-0017-11eb-8a79-0a7d87662d6a.png)

As you can see, in my main FF Decentraleyes has used local resources 328k times. 

Worst case, the data is 3 years, best case about 4 years
 - When FF57 came out in Nov 2017 (that's the release when legacy extensions were no more), or even a few releases earlier as extensions migrated/updated .. I know I clean installed a few of them and counters got reset. I don't think Decentraleyes was one of them: so my best guess is that the counter represents about an extra year

And I didn't exactly write anything down, but I have paid attention to it
- I am pretty sure (99%) I started the year in the 320 thousands
- I've watched it very closely the last 3 months and it moved about 100

In other words, decentraleyes' resources are so out of date that it now doesn't cover anything meaningful.

I don't give a shit about LocalCDN as a replacement - because FPI already isolates third parties, and possible speed perf is not a privacy issue - in fact the opposite - an extension injection resources is a security risk - feel free to do what you want, but I'm not recommending it
